### PR TITLE
Added comments back and added aliases

### PR DESCRIPTION
**Pull request by renatoi**
@src-code @thierryk

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/rules.js [hunk 1]
```diff
@@ -1699,6 +2163,289 @@ module.exports = [
...
+        }
+    },
+    // matrix3d
+    {
+        "type": "pattern",
+        "name": "Matrix 3d (transform)",
+        "matcher": "Matrix3d",
+        "allowParamToValue": true,
+        "styles": {
+            "transform": "matrix($0,$1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$13,$14,$15)"
```
Comment (src-code) [inline src/rules.js]: This one will probably look funny in the reference docs... you'll get 15 "value" strings in a row :-/

**Comment by src-code** [comment]
LGTM, but @thierryk should weigh in before we merge this

File: src/rules.js (hunk 1 above)
Comment (renatoi) [inline src/rules.js]: It's 16! :) I was wondering if I should remove matrix.

**Comment by thierryk** [comment]
Sorry @renatoi, you need to revert the [specificity thing] (https://github.com/yahoo/atomizer/commit/02f9ddb68d43810e727d1453d0e66aa4661c4633) change. My bad...

**After that**, LGTM, :shipit:

**Pull request merged**